app/coffee/WebApiManager.coffee

```diff
@@ -25,6 +26,10 @@ module.exports = WebApiManager =
 			if 200 <= response.statusCode < 300
 				callback null, data?.project, data?.privilegeLevel
 			else
-				err = new Error("non-success status code from web: #{response.statusCode}")
-				logger.error {err, project_id, user_id}, "error accessing web api" 
-				callback err
+				if response.statusCode == 429
```

Comment:
```suggestion
			else if response.statusCode == 429
```

and remove the indent - might be easier to read

Reply:
:+1: yeah, that's better